pkgs/by-name/ma/marcel/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, python3
+, bash
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "marcel";
+  version = "0.20.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "geophile";
+    repo = "marcel";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DlpjE0RTn2zdTkSKXGvuZXBk8R6ZKrhRe8U7xchyauo=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    dill
+    psutil
+    bash
+  ];
+
+  doCheck = false; # cause the tests provided by marcel will read/write $HOME/.local/share/marcel and /tmp and use sudo
+
+  postFixup = ''
+    substituteInPlace $out/bin/marcel \
+        --replace python3 ${python3}/bin/python3
+    wrapProgram $out/bin/marcel \
```

Comment:
This is strange.
Usually wrapProgram is a script available from makeWrapper package, but you are using it without the package...

Reply:
I believe that because normal python scripts need to be wrapped (for the pythonpath), makeWrapper is included when building a python package